Yoast/javascript
# Refactors the feature toggle package to make use of a global variable.

## Summary

* [not-user-facing] Sets the feature flags on the window / worker instead of a 'local' global variable.

## Relevant technical choices:

* Setting it on the window / worker (on `self`) makes it widely available on the javascript side of the plugin (e.g. also in other packages like `components`).

## Test instructions

This PR can be tested by following these steps:

* Add the following line to your `wp-config.php` to enable two (non-existing) features:
  `define( 'YOAST_SEO_ENABLED_FEATURES', 'some-feature, another-feature' );`
* Open a file in `components` that gets loaded on a post or page.
  * For example `KeywordSuggestions.js` or `Collapsible.js`.
* Add the following line in the render method or other method that is called inside of the metabox:
  * `console.log( "enabled features", enabledFeatures() );`.
* Open a post or page.
* Open the browser console.
* The console should show: `enabled features: ["some-feature", "another-feature"]`.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * 

## UI changes

## Quality assurance


Fixes #
